# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmic-comp/hotfix-libs.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cosmic-comp-epoch-1.0.0-alpha.5.1/build.rs.vanilla	2025-01-24 16:02:05.463333370 +0100
+++ cosmic-comp-epoch-1.0.0-alpha.5.1/build.rs	2025-01-24 16:02:42.680000040 +0100
@@ -9,5 +9,8 @@
     {
         let git_hash = String::from_utf8(output.stdout).unwrap();
         println!("cargo:rustc-env=GIT_HASH={}", git_hash);
+        println!("cargo:rustc-link-lib=xkbcommon");
+        println!("cargo:rustc-link-lib=pixman-1");
+        println!("cargo:rustc-link-search=native=/usr/X11R7/lib64");
     }
 }
